zuul.d/jobs.yaml

```diff
@@ -665,3 +665,15 @@
     override-checkout: v0.15.0
     secrets:
       - huaweicloud_credentials
+
+- job:
+    name: docker-machine-0.15.0-functional-devstack-mitaka
+    parent: init-test
+    description: |
+      Test image building functionality of Docker machine of 0.15.0 version against devstack of Mitaka.
+    run: playbooks/docker-machine-functional-devstack-mitaka/run.yaml
```

Comment:
We can reuse the job for all of OpenStack releases, so please do not append `mitaka` at job name, like packer job.

Reply:
because of the auth url problem this job cannot run with other releases of OpenStack, and also  the docker installation packages are specific for ubuntu trusty.

Reply:
We should show the result even if docker-machine can not work with these OpenStack release.